client/src/components/events/FindLocationForm.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+    <div />
+</template>
+<script>
+import * as i18nGetters from '@/store/modules/i18n/types/getters';
+import places from 'places.js';
+import { mapGetters } from 'vuex';
+import eventTypeMixin from '@/components/events/eventTypeMixin';
+
+const VUE_APP_ALGOLIA_API_KEY = process.env.VUE_APP_ALGOLIA_API_KEY;
+const VUE_APP_ALGOLIA_APP_ID = process.env.VUE_APP_ALGOLIA_APP_ID;
+
+export default {
+    name: 'FindLocationForm',
+    mixins: [eventTypeMixin],
+    data() {
+        return {
+            placesAutocomplete: null
+        };
+    },
+    beforeDestroy() {
+        this.placesAutocomplete.destroy();
+    },
+    computed: {
+        ...mapGetters('i18n', {
+            lang: i18nGetters.GET_LANGUAGE_CONSTANTS
+        })
+    },
+    mounted() {
+        this.input = document.createElement('input');
+        this.input.setAttribute('placeholder', this.lang.PICK_NEEDED_LOCATION);
+        this.input.setAttribute('type', 'search');
+        this.$el.appendChild(this.input);
+
+        this.placesAutocomplete = places({
+            appId: VUE_APP_ALGOLIA_APP_ID,
+            apiKey: VUE_APP_ALGOLIA_API_KEY,
+            container: this.input
+        });
+        this.placesAutocomplete.configure({
+            language: 'en'
```

Comment:
Does ALGOLIA have Ukrainian localization? It is question only for future in this stage of project English autocomplete will be enough

Reply:
yeah, can we remove language to use the default one (it will use browser's default)?

Reply:
You can get current language from state /store/modules/i18n/  see  getter [i18n.GET_CURRENT_LANGUAGE].
If exist autocomplete for ua using of this getter good solution. But as I mention before maybe we can stay only for english autocompletion